apps/el-gen/genesis_besu.py

```diff
@@ -115,17 +115,15 @@
         "timestamp": str(data['genesis_timestamp'])
     }
 
-    for key, value in data['el_premine'].items():
-        acct = w3.eth.account.from_mnemonic(data['mnemonic'], account_path=key, passphrase='')
-        weival = value.replace('ETH', '0' * 18)
-        out["alloc"][acct.address] = {"balance": weival}
-
-
         # Some hardcoded addrs
     for key, value in data['el_premine_addrs'].items():
         weival = value.replace('ETH', '0' * 18)
         out["alloc"][key] = {"balance": weival}
 
 out['config']['ethash'] =  {}
-out['config']['cancunTime'] = int(2000000000)
+if int(data['chain_id']) != 17000:
```

Comment:
sure you want the logic this way around? it'd make more sense to check if its 17000 and if so, set the cancunTime to a specified value